src/serialport.h

```diff
@@ -150,6 +150,9 @@ struct ListResultItem {
   std::string locationId;
   std::string vendorId;
   std::string productId;
+#ifdef UWP_DLL
+  std::string displayName;

```

Comment:
I'll take it out so it's consistent with other platforms. 
The displayName is supposed to be the friendly name for the device. It maps to the [System.ItemNameDisplay](https://msdn.microsoft.com/en-us/library/windows/desktop/bb760770(v=vs.85).aspx) Windows property. I thought it would be nice to have but it's not necessary.
So for example if I have a Leonardo attached to my machine and I run serialport.list, this is what I can get...

>comName: COM4
pnpId: USB\VID_2341&PID_8036&MI_00\7&1f38248&0&0000
manufacturer: undefined
displayName: Arduino Leonardo